lib/matplotlib/tests/test_mathtext.py

```diff
@@ -385,3 +385,35 @@ def test_math_fontfamily():
              size=24, math_fontfamily='dejavusans')
     fig.text(0.2, 0.3, r"$This\ text\ should\ have\ another$",
              size=24, math_fontfamily='stix')
+
+def test_default_math_fontfamily():
+    mpl.rcParams['mathtext.fontset'] = 'cm'
+    test_str = r'Foo: $\alpha_{i+1}^j = {\rm sin}(2\pi f_j t_i) e^{-5 t_i/\tau}$'
+
+    buff = io.BytesIO()
+    fig, ax = plt.subplots()
+
+    prop1 = fig.text(0.1, 0.1, test_str, font='Arial').get_fontproperties()
+    assert prop1.get_math_fontfamily() == 'cm'
+    prop2 = fig.text(0.1, 0.2, test_str, fontproperties='Arial').get_fontproperties()
```

Comment:
flake8 is ignoring them because of some of the test strings, but these lines are too long.

Reply:
Maybe break the `prop1` and `prop2` sentences into 2 lines.
```python3
text1 = fig.text(0.1, 0.1, test_str, font='Arial')
prop1 = text1.get_fontproperties()
```